docs/schema/1.0/manifest.schema.json

```diff
@@ -0,0 +1,164 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/getoutreach/clientxtsdk/docs/schema/1.0/manifest.schema.json",
+    "properties": {
+        "version": {
+            "type": "string",
+            "pattern": "^[0-9]+\\.[0-9]+$",
```

Comment:
add same requirement to docs/manifest.md

Reply:
done